packages/elements/src/components/ino-switch/ino-switch.scss

```diff
@@ -7,60 +7,62 @@
 $n-5: #abb1cc;
 $n-6: #cbd0e0;
 
-@mixin set-track-border-color($color) {
+/* colors */
+$primary: #2c02ff;
+$focus: #2d5dff4d;
```

Comment:
I would suggest copying the `_new-theme.scss` from here https://github.com/inovex/elements/pull/712/files#diff-b6776d33e431be01d3f60db81bb96bd67c5766209cc39a44a003594c9b3fd943 and then using them instead of defining the colors again.